server/server.go

```diff
@@ -1,15 +1,42 @@
 package main
 
 import (
+	"net/http"
+
 	"github.com/labstack/echo/v4"
 )
 
+type ContentResponseMeta struct {
+	Total int `json:"total"`
+}
+
+type Content struct {
+	Body string `json:"body"`
+}
+
+type Response struct {
```

Comment:
You'd probably want to call this a ContentResponse since all endpoints will have responses.